### DISCUSSION TITLE
How to add tailwindcss in nextJs project?

### DISCUSSION THREAD

**Discussion by azimuddin017** [community/community#67946] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

How to add tailwindcss in nextJs project?

**Comment by web-emran** [comment]
1. Install: npm install sass --save-dev
2. Import: import '../scss/yourStyle.scss';

But I don't think you need SCSS when you use TailwindCSS though. TailwindCSS is very powerful

**Comment by queenofcorgis** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
